.eslintrc

```diff
@@ -73,42 +79,51 @@
       }
     ],
 
-    // TODO: evaluate and potentially turn these rules back on
-    "react/no-string-refs": 0, // TODO(awong): Enable.
-    "react/prop-types": 0, // TODO(awong): Enable.
-    "react/sort-prop-types": [
-      0,
-      { "callbacksLast": true, "requiredFirst": true }
-    ], // TODO(awong): Too hard to turn on.
+    /* ----- TODO: evaluate and potentially turn these rules back on ----- */
+
+    /* || Eslint main rules || */
+    "no-plusplus": 0,
+    "no-negated-condition": 0, // Sometimes negated conditions are easier to understand.
+    "no-underscore-dangle": 0, // We have build flags that use this
     "class-methods-use-this": 0,
     "global-require": 0,
     "lines-around-directive": 0,
     "no-mixed-operators": 0,
     "no-undef-init": 0,
     "padded-blocks": 0,
+
+    /* || import plugin || */
+    "import/no-extraneous-dependencies": 0,
     "import/extensions": 0,
     "import/first": 0,
     "import/no-named-as-default": 0,
     "import/prefer-default-export": 0,
+
+    /* || jsx-a11y plugin || */
     "jsx-a11y/href-no-hash": 0,
     "jsx-a11y/no-static-element-interactions": 0,
+
+    /* || react plugin || */
+    "react/no-string-refs": 0, // TODO(awong): Enable.
+    "react/prop-types": 0, // TODO(awong): Enable.
```

Comment:
We can turn on these two to see locally how many places need to have string refs refactored or prop types added, but if there are too many files, we should probably deprioritize. I think the other prop type rules would catch too many instances, but I haven't checked.